### ISSUE TITLE
Apply for unsplash full data.

### ISSUE THREAD

**Issue by xiao2mo** [unsplash/datasets#59]
Hi, Can u boost the application process for this dataset.
Really eager for some experiments in related research.
I found it's both hard for research or personal-use application.
I wonder how to apply for the full dataset ??
My email is checkmate.ming@gmail.com many thanks.

**Comment by xiao2mo** [comment]
for emergency use. +++++++

**Comment by xiao2mo** [comment]
wait for ur response!! Help

**Comment by wonjunior** [comment]
Same with me, I contacted them three times already but they have not replied yet!

**Comment by wonjunior** [comment]
This leads me to believe the repo is not maintained anymore and the dataset is no longer available.

**Comment by yzfly** [comment]
Same with me, I wonder the dataset is still available?

**Comment by aiXia121** [comment]
have someone got this dataset? 
thanks

**Comment by TimmyCarbone** [comment]
Hi everyone, the dataset's next release is scheduled for March.

You can request access to the Full Dataset here:
https://unsplash.com/data

I'll let the team responsible for those requests know that some people seem to be waiting on a reply.

**Comment by wonjunior** [comment]
Yes, please inform us when this new version will be released. Thank you.

**Comment by zijieloooooou** [comment]
> Hi everyone, the dataset's next release is scheduled for March.
> 
> You can request access to the Full Dataset here: https://unsplash.com/data
> 
> I'll let the team responsible for those requests know that some people seem to be waiting on a reply.

@TimmyCarbone 
May I use images from Unsplash to organize a non-commercial academic competition? Would doing so entail any legal risks?

**Comment by TimmyCarbone** [comment]
@zijieloooooou I believe it should be fine to use it if it's for academic/research purposes.

Note that it would still depend on the nature of the work that the contestants will do. For example, they will be exposed to legal risks if they feed the images to a commercial enterprise (like a 3rd party AI/LLM).

The people reviewing the requests should be able to help you make sure you're within your rights.

**Comment by zijieloooooou** [comment]
> [@zijieloooooou](https://github.com/zijieloooooou) I believe it should be fine to use it if it's for academic/research purposes.
> 
> Note that it would still depend on the nature of the work that the contestants will do. For example, they will be exposed to legal risks if they feed the images to a commercial enterprise (like a 3rd party AI/LLM).
> 
> The people reviewing the requests should be able to help you make sure you're within your rights.

Thank you so much for your response. I would like to confirm that this competition is strictly non-commercial and focused solely on the academic evaluation of composition. We will strictly limit access to the images and ensure they are used only within the context of the competition’s aesthetic analysis. We will explicitly instruct all participants that the images must not be fed into any third-party commercial AI/LLM tools or used for any commercial enterprise. We highly value the work of the Unsplash community and want to make sure we stay within our rights. If you have any further specific requirements or if there are any other concerns, please feel free to reach out to me directly at: lzj7@meitu.com.
